specificatie/openapi.yaml

```diff
@@ -648,6 +810,115 @@ components:
               items:
                 $ref: '#/components/schemas/ZoekResultaatHal'
 
+    GeoJsonZoekOpdracht:
+      description: 'Zoek opdracht voor het zoeken naar resource binnen een geometrie.'
+      type: object
+      required:
+        - geometrie
+        - spatialOperator
```

Comment:
spatialOperator heeft een default. Waarom is die dan toch required?

Reply:
pas ik aan